### DISCUSSION TITLE
Sublime Text IntelliSense

### DISCUSSION THREAD

**Discussion by YunshuaiPeng** [windicss/windicss#281] [Ideas]
Is there a plan to support sublime text?

**Comment by shiftgeist** [comment]
If you just want to get windicss running you can do so already. But my guess would be that you thought of a plugin for sublime which doesn't seem to exist.

For tailwind there is a plugin that has auto completion: https://github.com/bradlc/sublime-tailwindcss. Maybe this is a good base to build upon or is even so far configurable that it can be used for windicss.
